lib/mayaUsd/ufe/private/Utils.cpp

```diff
@@ -131,13 +131,20 @@ UsdGeomXformCommonAPI convertToCompatibleCommonAPI(const UsdPrim& prim)
 
 void applyCommandRestriction(const UsdPrim& prim, const std::string& commandName)
 {
+    // return early if prim is the pseudo-root.
+    // this is a special case and could happen when one tries to drag a prim under the 
+    // proxy shape in outliner. Also note if prim is the pseudo-root, no def primSpec will be found.
+    if (prim.IsPseudoRoot()){
+        return;
+    }
+
```

Comment:
@ppt-adsk per out offline discussion, PseudoRoot() is a special case and should be treated differently.

After some thinking, it makes more sense to simply return early if we are dealing with the root layer.